src/generate/generator.rs

```diff
@@ -105,6 +105,9 @@ impl Generator {
                     let mut path = path.clone();
                     path.push("target");
                     if path.exists() {
+                        path.push("generated");
+                        path.push(crate_name);
+                        let _ = std::fs::create_dir_all(&path);
```

Comment:
If this fails, we should return `false` immediately

Reply:
Done